hardhat.config.js

```diff
@@ -20,17 +20,19 @@ if (MAINNET_RPC_URL !== "https://eth-mainnet.alchemyapi.io/v2/your-api-key") set
 if (POLYGON_MAINNET_RPC_URL !== "https://polygon-mainnet.alchemyapi.io/v2/your-api-key") setRpcUrlCount++
```

Comment:
What's the point of all those different default strings? Just use one constant "UNSET" or something like that.

Reply:
Yeah they were a holdover from the original tooling repo, but I agree they don't make much sense.